packages/salesforcedx-vscode-apex/src/languageServer.ts

```diff
@@ -133,6 +139,14 @@ export async function createLanguageServer(
   extensionContext: vscode.ExtensionContext
 ): Promise<LanguageClient> {
   const server = await createServer(extensionContext);
+  // we no longer want to watch the StandardApexLibrary folder
+  vscode.workspace
```

Comment:
We do not want to include the standard apex class library in file watching. The files are immutable and are pre indexed.